docker-compose.yml

```diff
@@ -2,18 +2,21 @@ version: '3'
 
 services:
   chatserver:
-    build:
-      context: ./server
-      dockerfile: Dockerfile
+    build: ./server/
+    volumes:
+      - ./server:/usr/app
```

Comment:
So we can have hot reload.
Basically what's doing is looking at "./server" in our filesystem, the "host" and mapping to "/usr/app" which is the workdir. 
If we don't have that it won't persist our changes.